### ISSUE TITLE
Limit on click data on the manage page

### ISSUE THREAD

**Issue by nomadmtb** [nomadmtb/utldr#2]
If there is years of click data then it will try to load all of that data into the chart within the manage page.  This works for newer targets, but it will look really weird when there is a ton of data.  Maybe limit the chart to the last 30 days or so.  User always has the option to download the data too.

**Comment by nomadmtb** [comment]
```
[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]

len = 15
we want the last 30 items

if len > 30 

len - 30 = start_point
```

**Comment by nomadmtb** [comment]
Fixed with [8d6d025](https://github.com/nomadmtb/utldr/commit/8d6d025d7622e3db461aa12578a19497a8a4028d)
